plugins/event_source/journald.py

```diff
@@ -0,0 +1,62 @@
+"""
+journald.py
+An ansible-events event source plugin that tails systemd journald logs.
+
+Arguments:
+    match - return messages that matches this field, see https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html
+
+Examples:
+    - name: Return severity 6 messages
+      ansible.eda.journald:
+        match: "PRIORITY=6"
+
+    - name: Return messages when sudo is used
+      become: true
```

Comment:
I am not sure the source plugin supports `become: true`. Did you test?

Reply:
I'm pretty sure I did a month ago, will retest.

Reply:
my bad, it fails